weblate/trans/tests/test_acl.py

```diff
@@ -122,7 +122,7 @@ def test_invite_invalid(self):
             follow=True,
         )
         # This error comes from Django validation
-        self.assertContains(response, "Enter a valid email addres")
+        self.assertContains(response, "Enter a valid e-mail address")
```

Comment:
Changing assertion without changing the actual message breaks testsuite. As described by the comment, the error comes from Django here.

Reply:
Found the error when I loaded githubusercontent.
Would make sense to me if weblate/weblate/trans/views/acl.py not had any such message to return.
https://hosted.weblate.org/translate/weblate/website/en/?checksum=f495c8cb6bf18bc9 is the changed statement, albeit in different files, with an ending dot (which I think is fine).
Is it in a different repo? Will I find the location in the file? Do I have to read https://docs.djangoproject.com/en/2.2/topics/testing/tools/#django.test.SimpleTestCase.assertContains

Reply:
This one is at https://github.com/django/django/blob/fd999318ad78613227cdb7c5656345d9e216802b/django/core/validators.py#L161